Makefile

```diff
@@ -821,23 +849,17 @@ else
 endif
 
 $(ROM): $(ELF)
-	$(ELF2ROM) -cic $(CIC) $< $@
+	$(OBJCOPY) --pad-to 0x$$($(OBJDUMP) -t $< | grep _RomSize | cut -d ' ' -f 1) -O binary $< $@
+	$(PYTHON) -m ipl3checksum sum --cic $(CIC) --update $@
```

Comment:
woooo, happy to see we are finally ditching elf2rom.
btw, that objdump invocation seems a bit convoluted, it may be nice to have a comment about why and what it is doing.